### ISSUE TITLE
update screen after pop

### ISSUE THREAD

**Issue by Farg0k** [Milad-Akarie/auto_route_library#2116]
Need this simple function

**Comment by gidrokolbaska** [comment]
use callback as a parameter of a popped screen or use PopScope widget

**Comment by Farg0k** [comment]
I don't know how to do it. 
Update screen after pop is such a big problem that this is not included in the package?

**Comment by gidrokolbaska** [comment]
> I don't know how to do it. Update screen after pop is such a big problem that this is not included in the package?

Yes, it's not included because the purpose of the package is navigation... Try to google what is callback and how to provide a callback function to your screen. Also, pushing a new screen is a Future. Await it and after the screen is popped, further code is executed. Example:
await context.router.push(YouScreenRoute()); 
updateTheDataFunction(); 

In this example updateTheDataFunction() will be called after you pop from YouScreenRoute. 
Another approach is a callback function (google it yourself)

**Comment by Farg0k** [comment]
Thank you for your help
